docs/how-to/h-deploy-eks.md

```diff
@@ -0,0 +1,287 @@
+# How to deploy on EKS
+
+[Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS) is a popular, fully automated Kubernetes service. To access the EKS Web interface, go to [console.aws.amazon.com/eks/home](https://console.aws.amazon.com/eks/home).
+
+## Summary
+* [Install EKS and Juju tooling](#install-eks-juju)
+* [Create a new EKS cluster](#create-eks-cluster)
+* [Bootstrap Juju on EKS](#boostrap-juju)
+* [Deploy charms](#deploy-charms)
+* [Display deployment information](#display-information)
+* [Clean up](#clean-up)
+
+---
+
+## Install EKS and Juju tooling
+
+Install [Juju](https://juju.is/docs/juju/install-juju) and the [`kubectl` CLI tools](https://kubernetes.io/docs/tasks/tools/) (that will be used for managing the Kubernetes cluster) via snap:
+
+```shell
+sudo snap install juju --channel 3.5/stable
+sudo snap install kubectl --classic
+```
+
+
+
+Follow the installation guides for:
+* [eksctl](https://eksctl.io/installation/) - the Amazon EKS CLI
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) - the Amazon Web Services CLI
+
+To check they are all correctly installed, you can run the commands below.
+
+```shell
+juju version
+```
+
+[details="Sample output:"]
+```shell
+3.5.2-genericlinux-amd64
+```
+[/details]
+
+```shell
+kubectl version --client
+``` 
+
+[details="Sample output:"]
+```shell
+Client Version: v1.28.2
+Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
+```
+[/details]
+
+```shell
+eksctl info
+```
+
+[details="Sample output:"]
+```shell
+eksctl version: 0.159.0
+kubectl version: v1.28.2
+```
+[/details]
+
+```shell
+aws --version
+```
+
+[details="Sample output:"]
+```shell
+aws-cli/2.13.25 Python/3.11.5 Linux/6.2.0-33-generic exe/x86_64.ubuntu.23 prompt/off
+```
+[/details]
+
+### Authenticate
+
+Create an IAM account (or use legacy access keys) and login to AWS:
+
+```shell
+> aws configure
```

Comment:
Not a biggie, but can we fix the usage of the prompt for combining input and output here, since we do it everywhere else? As far as I can see this is the last place to fix it (remove the `>` and split output from input with proper introduction.

Reply:
I actually purposely didn't want to do that since it is not JUST output. The lines below should also provide guidance on how to feed the inputs (although I know it is obvious), and we actually use placeholder instead of just taking the output of a screen